assets/src/scripts/routes/common.js

```diff
@@ -108,14 +108,52 @@ export default {
 				// Assign the button
 				let btn = heading.querySelector( 'button' );
 
-				btn.onclick = () => {
+				let listItems = content.children;
+
+				// Handle list items and events
+				Array.prototype.forEach.call( listItems, item => {
+					// Click the anchor tag inside the list item if the list item is clicked.
+					item.onclick = () => {
+						item.firstElementChild.click();
+					};
+
+					// Click the anchor tag inside the list item if the list item is active and 'Enter' is pressed.
+					item.onkeydown = e => {
+						if ( e.which === 13 ) {
+							item.firstElementChild.click();
+						}
+					};
+
+					// Collapse the content menu if user tabs out.
+					item.onblur = e => {
+						if ( item === listItems[listItems.length - 1] && e.relatedTarget.nodeName !== 'LI' ) {
```

Comment:
This is arguably a little hacky, but was what I could come up with in a pinch.  Basically, if this is a) the last item in the list, and b) the element that you're tabbing too isn't a list item, then we can proceed to close the menu.  The last condition (checking for a list item) is so the user can shift + tab from the last list item to the item before it without the menu collapsing.